src/components/Icons/IconTypes.tsx

```diff
@@ -1,9 +1,9 @@
 /* eslint-disable camelcase */
 export enum IconRotationTypes {
-  rotate180 = "rotate-180",
-  rotate270 = "rotate-270",
   rotate0 = "rotate-0",
   rotate90 = "rotate-90",
+  rotate180 = "rotate-180",
+  rotate270 = "rotate-270",
 }
 
 export enum IconNames {
```

Comment:
Just wondering if this enum is necessary given that we already have the allSvgs object declared in icon.tsx.  It might be necessary--it just flagged my attention so I though it worth mentioning.

Reply:
@kaplanmichael 
I think the names were used in one of the stories - a story that I am now omitting in the `.mdx` version of the stories.  I am omitting the story becaseu I cann't figure out a way to redo what was being done in the `.tsx` version.  Do you think it is necessary to include a story that shows all available icons at one time?